site/public/css/diff-viewer.css

```diff
@@ -1,4 +1,3 @@
-@import url("reset.css");
```

Comment:
just doing this then makes the popup look ugly though since you're still bringing sever.css into it which you most likely don't want to do (and why you added the reset.css in the first place.